examplePlugins/hybrid/v1/metadata.json

```diff
@@ -27,9 +28,9 @@
         "types"
     ],
     "actions": {
-        "__testConfig":        { "connector": "NodeJS", "version": "1.6.0", "config": { "scriptPath": "handler.js", "entryPoint": "testConfig" } },
-        "import":              { "connector": "NodeJS", "version": "1.6.0", "config": { "scriptPath": "handler.js", "entryPoint": "importObjects" }, "isImportAction": true },
-        "datasourceUnscoped":  { "connector": "NodeJS", "version": "1.6.0", "config": { "scriptPath": "handler.js", "entryPoint": "readDataSource" } },
-        "appScopedProperties": { "connector": "NodeJS", "version": "1.6.0", "config": { "scriptPath": "handler.js", "entryPoint": "readDataSource" } }
+        "__testConfig":        { "connector": "nodejs", "version": "2.0.0", "config": { "scriptPath": "handler.js", "entryPoint": "testConfig" } },
```

Comment:
Just checking that the casing change in 'nodejs' is deliberate here? It may not make any difference but thought I should check

Reply:
This is an important change which is fixing a missing "action" in `metadata.json` - i.e. *exactly* what my fix to `validate.js` in [PR #909 - Get validate script to police the actions for on-prem](https://github.com/squaredup/squaredup-plugin-repository/pull/909) will stop us from doing again :)

Reply:
All the real plugins in the private repo use all lower-case here - I'm not sure if it's case-sensitive or not TBH - safe to use what we use internally, though.